### ISSUE TITLE
another key_mask minor bug 

### ISSUE THREAD

**Issue by bound2020** [kang205/SASRec#17]
https://github.com/kang205/SASRec/blob/d0b823a1b2845ea90b090eaf6d3386810005b83a/modules.py#L185
tf.abs has no axis parameter, should be `tf.reduce_sum(tf.abs(keys), axis=-1)`

**Comment by pmixer** [comment]
Thx for pointing it out! Sorry I didn't do the necessary testing before sending the PR, would get it updated soon!

**Comment by kang205** [comment]
Thanks for pointing out!
